libsass bindings not found

Using Node v4.1.1

```
$ gulp
/home/dave/code/jspm-react/node_modules/gulp-sass/node_modules/node-sass/lib/index.js:22
    throw new Error('`libsass` bindings not found. Try reinstalling `node-sass`?');
    ^

Error: `libsass` bindings not found. Try reinstalling `node-sass`?
    at getBinding (/home/dave/code/jspm-react/node_modules/gulp-sass/node_modules/node-sass/lib/index.js:22:11)
    at Object.<anonymous> (/home/dave/code/jspm-react/node_modules/gulp-sass/node_modules/node-sass/lib/index.js:188:23)
    at Module._compile (module.js:434:26)
    at Module._extensions..js (module.js:452:10)
    at Object.require.extensions.(anonymous function) [as .js] (/home/dave/code/jspm-react/node_modules/babel/node_modules/babel-core/lib/api/register/node.js:214:7)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/home/dave/code/jspm-react/node_modules/gulp-sass/index.js:3:17)
```

When trying to rebuild:

```
$ npm rebuild node-sass

> node-sass@3.3.3 install /home/dave/code/jspm-react/node_modules/node-sass
> node scripts/install.js


> node-sass@3.3.3 postinstall /home/dave/code/jspm-react/node_modules/node-sass
> node scripts/build.js

` /home/dave/code/jspm-react/node_modules/node-sass/vendor/linux-x64-46/binding.node ` exists. 
 testing binary.
Binary is fine; exiting.

> node-sass@2.1.1 install /home/dave/code/jspm-react/node_modules/gulp-sass/node_modules/node-sass
> node scripts/install.js


> node-sass@2.1.1 postinstall /home/dave/code/jspm-react/node_modules/gulp-sass/node_modules/node-sass
> node scripts/build.js

make: Entering directory '/home/dave/code/jspm-react/node_modules/gulp-sass/node_modules/node-sass/build'
  CXX(target) Release/obj.target/binding/src/binding.o
In file included from ../src/binding.cpp:1:0:
../node_modules/nan/nan.h:324:27: error: redefinition of ‘template<class T> v8::Local<T> Nan::imp::NanEnsureHandleOrPersistent(const v8::Local<T>&)’
   NAN_INLINE v8::Local<T> NanEnsureHandleOrPersistent(const v8::Local<T> &val) {
                           ^
../node_modules/nan/nan.h:319:17: note: ‘template<class T> v8::Handle<T> Nan::imp::NanEnsureHandleOrPersistent(v8::Handle<T>&)’ previously declared here
   v8::Handle<T> NanEnsureHandleOrPersistent(const v8::Handle<T> &val) {
                 ^
../node_modules/nan/nan.h:344:27: error: redefinition of ‘template<class T> v8::Local<T> Nan::imp::NanEnsureLocal(v8::Handle<T>&)’
   NAN_INLINE v8::Local<T> NanEnsureLocal(const v8::Handle<T> &val) {
                           ^
../node_modules/nan/nan.h:334:27: note: ‘template<class T> v8::Local<T> Nan::imp::NanEnsureLocal(const v8::Local<T>&)’ previously declared here
   NAN_INLINE v8::Local<T> NanEnsureLocal(const v8::Local<T> &val) {
                           ^
../node_modules/nan/nan.h:757:13: error: ‘node::smalloc’ has not been declared
     , node::smalloc::FreeCallback callback
             ^
../node_modules/nan/nan.h:757:35: error: expected ‘,’ or ‘...’ before ‘callback’
     , node::smalloc::FreeCallback callback
                                   ^
../node_modules/nan/nan.h: In function ‘v8::Local<v8::Object> NanNewBufferHandle(char*, size_t, int)’:
../node_modules/nan/nan.h:761:50: error: ‘callback’ was not declared in this scope
         v8::Isolate::GetCurrent(), data, length, callback, hint);
                                                  ^
../node_modules/nan/nan.h:761:60: error: ‘hint’ was not declared in this scope
         v8::Isolate::GetCurrent(), data, length, callback, hint);
                                                            ^
../node_modules/nan/nan.h: In function ‘v8::Local<v8::Object> NanNewBufferHandle(const char*, uint32_t)’:
../node_modules/nan/nan.h:768:67: error: call of overloaded ‘New(v8::Isolate*, const char*&, uint32_t&)’ is ambiguous
     return node::Buffer::New(v8::Isolate::GetCurrent(), data, size);
                                                                   ^
../node_modules/nan/nan.h:768:67: note: candidates are:
In file included from ../node_modules/nan/nan.h:25:0,
                 from ../src/binding.cpp:1:
/home/dave/.node-gyp/4.1.1/src/node_buffer.h:31:40: note: v8::MaybeLocal<v8::Object> node::Buffer::New(v8::Isolate*, v8::Local<v8::String>, node::encoding) <near match>
 NODE_EXTERN v8::MaybeLocal<v8::Object> New(v8::Isolate* isolate,
                                        ^
/home/dave/.node-gyp/4.1.1/src/node_buffer.h:31:40: note:   no known conversion for argument 3 from ‘uint32_t {aka unsigned int}’ to ‘node::encoding’
/home/dave/.node-gyp/4.1.1/src/node_buffer.h:43:40: note: v8::MaybeLocal<v8::Object> node::Buffer::New(v8::Isolate*, char*, size_t) <near match>
 NODE_EXTERN v8::MaybeLocal<v8::Object> New(v8::Isolate* isolate,
                                        ^
/home/dave/.node-gyp/4.1.1/src/node_buffer.h:43:40: note:   no known conversion for argument 2 from ‘const char*’ to ‘char*’
In file included from ../src/binding.cpp:1:0:
../node_modules/nan/nan.h: In function ‘v8::Local<v8::Object> NanNewBufferHandle(uint32_t)’:
../node_modules/nan/nan.h:772:61: error: could not convert ‘node::Buffer::New(v8::Isolate::GetCurrent(), ((size_t)size))’ from ‘v8::MaybeLocal<v8::Object>’ to ‘v8::Local<v8::Object>’
     return node::Buffer::New(v8::Isolate::GetCurrent(), size);
                                                             ^
../node_modules/nan/nan.h: In function ‘v8::Local<v8::Object> NanBufferUse(char*, uint32_t)’:
../node_modules/nan/nan.h:779:12: error: ‘Use’ is not a member of ‘node::Buffer’
     return node::Buffer::Use(v8::Isolate::GetCurrent(), data, size);
            ^
../src/binding.cpp: In function ‘void dispatched_async_uv_callback(uv_async_t*)’:
../src/binding.cpp:70:35: warning: ‘void node::FatalException(const v8::TryCatch&)’ is deprecated (declared at /home/dave/.node-gyp/4.1.1/src/node.h:282): Use FatalException(isolate, ...) [-Wdeprecated-declarations]
     node::FatalException(try_catch);
                                   ^
../src/binding.cpp: In function ‘void make_callback(uv_work_t*)’:
../src/binding.cpp:245:35: warning: ‘void node::FatalException(const v8::TryCatch&)’ is deprecated (declared at /home/dave/.node-gyp/4.1.1/src/node.h:282): Use FatalException(isolate, ...) [-Wdeprecated-declarations]
     node::FatalException(try_catch);
                                   ^
../src/binding.cpp: In function ‘void ImportedCallback(const v8::FunctionCallbackInfo<v8::Value>&)’:
../src/binding.cpp:365:35: warning: ‘void node::FatalException(const v8::TryCatch&)’ is deprecated (declared at /home/dave/.node-gyp/4.1.1/src/node.h:282): Use FatalException(isolate, ...) [-Wdeprecated-declarations]
     node::FatalException(try_catch);
                                   ^
binding.target.mk:128: recipe for target 'Release/obj.target/binding/src/binding.o' failed
make: *** [Release/obj.target/binding/src/binding.o] Error 1
make: Leaving directory '/home/dave/code/jspm-react/node_modules/gulp-sass/node_modules/node-sass/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/dave/code/jspm-react/node_modules/gulp-sass/node_modules/node-sass/node_modules/pangyp/lib/build.js:272:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Linux 3.19.0-15-generic
gyp ERR! command "/apps/node-v4.1.1-linux-x64/bin/node" "/home/dave/code/jspm-react/node_modules/gulp-sass/node_modules/node-sass/node_modules/pangyp/bin/node-gyp" "rebuild"
gyp ERR! cwd /home/dave/code/jspm-react/node_modules/gulp-sass/node_modules/node-sass
gyp ERR! node -v v4.1.1
gyp ERR! pangyp -v v2.3.2
gyp ERR! not ok 
Build failed
node-sass@3.3.3 /home/dave/code/jspm-react/node_modules/node-sass
node-sass@2.1.1 /home/dave/code/jspm-react/node_modules/gulp-sass/node_modules/node-sass
```

--- a/package.json
+++ b/package.json
@@ -24,7 +24,7 @@
     "gulp-jspm": "^0.3.1",
     "gulp-load-plugins": "^0.8.0",
     "gulp-rename": "^1.2.0",
-    "gulp-sass": "^1.2.4",
+    "gulp-sass": "^2.1.0",
     "gulp-shell": "^0.2.11",
     "gulp-sourcemaps": "^1.3.0",
     "gulp-uglify": "^1.0.2",

